src/main/scala/ai/privado/languageEngine/java/tagger/collection/CollectionTagger.scala

```diff
@@ -47,28 +46,17 @@ class CollectionTagger(cpg: Cpg, ruleCache: RuleCache) extends PrivadoParallelCp
     ruleCache.getRule.collections.filter(_.catLevelTwo == Constants.annotations).toArray
 
   override def runOnPart(builder: DiffGraphBuilder, ruleInfo: RuleInfo): Unit = {
```

Comment:
Refactored into smaller functions so that Kotlin tagger can re-use this class. All tests are passing.